### PR TITLE
add Ansible remediation and improve tests for audit_rules_networkconfig_modification

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

Add Ansible remediation.
Add tests for auditctl rules and tests with only partial configuration.

#### Rationale:

CIS productization efford

### CODE REVIEW

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# reboot = true
```
[the remark this answers is not in the document]
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/ansible/shared.yml:2]: I think it is not, just an oversight.

**Comment by vojtapolasek** [comment]
I also modified the rule to use not yet merged version of the macro. The macro is here: https://github.com/ComplianceAsCode/content/pull/5723

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/ansible/shared.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jan-cerny) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/ansible/shared.yml]: OK

**Comment by jan-cerny** [comment]
@vojtapolasek The new macro (#5273) has been merged so please rebase this PR on top of master.

**Comment by vojtapolasek** [comment]
rebased

**Pull request merged**